UiPath.FreeRdpClient/UiPath.FreeRdpWrapper/FreeRdpWrapper.cpp

```diff
@@ -219,7 +228,9 @@ namespace FreeRdpClient
 		return 0;
 	}
 
-	instance_data* transport_start(rdpContext* context, ConnectOptions* rdpOptions)
+	instance_data* transport_start(
```

Comment:
I think transport_start should not return instance_data because the instance can be deleted when the started thread is finished. It can return a BOOL and eventName as a reference parameter.